addons/escoria-core/game/core-scripts/esc/commands/set_active_if_exists.gd

```diff
@@ -0,0 +1,43 @@
+# `set_active_if_exists object active`
+#
+# Changes the "active" state of the object in the current room.
+# Inactive objects are invisible in the room.
```

Comment:
I made it to solve an internal problem. Whether a user would actually use this, I can't say. The only difference is that if the object being acted upon isn't already in the object manager, it won't error and will just do nothing.

Reply:
If it's an internal problem, does it even need a ESC script command? Can it just be called by internal function calls?
My thought is that if it needs to be in an ESC script, it should be clear to users when they would (or wouldn't) use it to make Escoria as user-friendly as possible.

Reply:
Unfortunately, yes, it does need to be an ESC script command because the loading system works based on using pre-compiled ESC commands via the event manager. This is actually what started that discussion a couple weeks ago about 'internal ESC commands", but that got shelved. I'm really not sure of a better idea of how else to tackle this, as we can't mix Escoria internal GDScript and ESC script commands. I'm open to further discussion/ideas, though.